app/components/sidebars/drawer_layout.js

```diff
@@ -196,14 +198,16 @@ export default class DrawerLayout extends Component {
             left: drawerPosition === 'left' ? 0 : null,
             right: drawerPosition === 'right' ? 0 : null,
         };
-
         /* Drawer styles */
         let outputRange;
+        // ios main sidebar sits mostly under the main screen, with slight move
+        const translateDistance = Platform.OS === 'ios' && isMain ?
+            Math.floor(drawerWidth * .2) : drawerWidth
```

Comment:
Not that it changes anything but it makes it more readable.

```suggestion
            Math.floor(drawerWidth * 0.2) : drawerWidth
```